eyelid-match-ops/src/primitives/poly/modular_poly/conf.rs

```diff
@@ -35,11 +35,9 @@ impl PolyConf for IrisBits {
 /// Full resolution polynomial parameters.
 ///
 /// These are the parameters for full resolution, according to the Inversed Tech report.
-#[cfg(not(tiny_poly))]
```

Comment:
The `encoded` module is only defined for `FullRes`. To support `tiny` or configurable sizes, we must unify all parameters (`IRIS_BIT_LENGTH`, etc) in the same configuration mechanism.

Reply:
The `tiny_poly` tests should pass after merging #79

Reply:
> The `tiny_poly` tests should pass after merging #79

I just merged both those PRs now.

Reply:
> Until then we can do tests on FullRes rather than TestRes.

That alone did not work before #79 because FullRes got the tiny Coeff, which cannot hold the sums.

Support for other resolutions is going to be another issue yes.

Reply:
> The `encoded` module is only defined for `FullRes`. To support `tiny` or configurable sizes, we must unify all parameters (`IRIS_BIT_LENGTH`, etc) in the same configuration mechanism.

This is now ticket #84.